test/App/Fossa/ManualDepsSpec.hs

```diff
@@ -54,6 +56,10 @@ theWorks = ManualDependencies references customs vendors remotes
       [ VendoredDependency "vendored" "path" Nothing
       , VendoredDependency "versioned" "path/to/dep" (Just "2.1.0")
       ]
+    locators =
+      [ Locator "fetcher-1" "one" Nothing
+      , Locator "fetcher-2" "two" $ Just "1.0.0"
+      ]
```

Comment:
[optional] This is an invalid locator, and if support accidentally provides an invalid locator to import it may be confusing as to why it isn't working.

Consider validating the provided fetcher using `fetcherToDepType`, such that if the result is `Nothing` FOSSA CLI emits a `logWarn` that the provided locator may not be valid.

Note: I don't think the contract of `Locator`'s `FromJSON` instance should change to do this, I think this should be special-cased in `ManualDeps`.